Multisig/UI/ClaimToken/Select Guardian/GuardianDetailsViewController.swift

```diff
@@ -57,7 +53,7 @@ class GuardianDetailsViewController: UIViewController {
 
         continueButton.setText("Select & Continue", .filled)
 
-        if (guardianAddress == Address.zero) {
+        if (guardian.address == Address.zero) {
```

Comment:
fyi, the `()` not needed in Swift's `if` statements

Reply:
old habits